CountToMillion/main.cpp

```diff
@@ -0,0 +1,25 @@
+#include <string>
```

Comment:
Insert a blank line above this line

The order of inclusions should be as follows:
1. Respective header ("WinDirStat.h" for "WinDirStat.cpp")
2. Standard headers (\<iostream>)
3. Relative headers ("MyFile.h")

In between each section should be a blank line.

- [ ] Conventions